src/style.css

```diff
@@ -0,0 +1,100 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 700;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.main {
+  margin: 0;
+  padding: 0;
+}
+
+.search {
+  color: #3d4e61;
+  font-weight: 300;
+  margin: 20px 8px;
+  position: relative;
+}
+
+.big-search {
+  width: 100%;
+  height: 70px;
+  padding-left: 62px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+
+  color: #3d4e61;
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+  font-weight: 300;
+  line-height: 22px;
+}
```

Comment:
It's better to set default font-family to html tag and use font-family: inherit for inputs